straxen/plugins/peaklet_processing.py

```diff
@@ -153,22 +153,44 @@ def compute(self, records, start, end):
         # Get hits outside peaklets, and store them separately.
         # fully_contained is OK provided gap_threshold > extension,
         # which is asserted inside strax.find_peaks.
-        lone_hits = hits[strax.fully_contained_in(hits, peaklets) == -1]
+        is_lone_hit = strax.fully_contained_in(hits, peaklets) == -1
+        lone_hits = hits[is_lone_hit]
         strax.integrate_lone_hits(
             lone_hits, records, peaklets,
             save_outside_hits=(self.config['peak_left_extension'],
                                self.config['peak_right_extension']),
             n_channels=len(self.to_pe))
 
         # Compute basic peak properties -- needed before natural breaks
-        strax.sum_waveform(peaklets, r, self.to_pe)
+        hits = hits[~is_lone_hit]
+        # Define regions outside of peaks such that _find_hit_integration_bounds
+        # is not extended beyond a peak.
+        outside_peaks = self.create_outside_peaks_region(peaklets, start, end)
+        strax.find_hit_integration_bounds(hits,
+                                          outside_peaks,
+                                          records,
+                                          (self.config['peak_left_extension'],
+                                           self.config['peak_right_extension']),
+                                           len(self.to_pe),
+                                          allow_bounds_beyond_records=True,
+                                          )
+
+        # Updated time and length of hits and sort again:
+        hits_time_shift = (hits['left'] - hits['left_integration']) * hits['dt']
+        hits['time'] = hits['time'] - hits_time_shift
+        hits['length'] = (hits['right_integration'] - hits['left_integration'])
+        hits = strax.sort_by_time(hits)
+        rlinks = strax.record_links(records)
```

Comment:
Not sure if there is much we can do about it. But like 0bbad6c this feels like we could hit issues with this changing definition. Maybe we can leave it for now unless you see some easy way of making this more explicit.

Reply:
We could starting calling these objects `hitlets` instead of hits? Then it would be consistent and the difference should be clear. Hit refers to the central part above threshold a `hitlet` to the entire signal including the left and right extension.

Reply:
sounds good 👍 , adding `Hit refers to the central part above threshold a hitlet to the entire signal including the left and right extension.` as a comment will help reminding the people less used to veto-processing

Reply:
Do you thin kit is fine: https://github.com/XENONnT/straxen/pull/601/commits/bc7df5e15275fc0e2a908951de578d2260055b6e ?